private/scribble-util.rkt

```diff
@@ -0,0 +1,24 @@
+#lang racket/base
```

Comment:
The name `scribble-util` isn't very descriptive. Could we call this module something like `scribble-cross-document-tech`, or something similar?